### PR TITLE
LeCongTu_BaiTap01

### PR DESCRIPTION
**Pull request by mvn-tule2-dn**
||

### CODE REVIEW

File: 001/TuLe/BaiTap1/BaiTap1/ContentView.swift [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+
+struct ContentView: View {
+
+    var colors: [Color] = [.red, .blue, .black, .orange]
+    @State var backgroundColor: Color = .white
+
+    var body: some View {
+        VStack {
+            Button {
+                guard let element = colors.randomElement() else {
```
Comment (mvn-tienle-dn) [inline 001/TuLe/BaiTap1/BaiTap1/ContentView.swift:18]: I need a default value for `element`, if this line is error.

**Review by mvn-tienle-dn — COMMENTED** [review]
Please, check & fix my comment

File: 001/TuLe/BaiTap1/BaiTap1/ContentView.swift (hunk 1 above)
Comment (mvn-tule2-dn) [reply 001/TuLe/BaiTap1/BaiTap1/ContentView.swift:18]: > I need a default value for `element`, if this line is error.

```swift
guard let element = colors.randomElement() else {
                    backgroundColor = .red
                    return
                }
```
I fixed it. is that right?

**Review by mvn-tienle-dn — COMMENTED** [review]
This is not a good solution.

**Review by mvn-tienle-dn — APPROVED** [review]
LGTM

**Pull request merged**